src/pages/builders/index.ts

```diff
@@ -384,3 +406,167 @@ export class CypressScriptBuilder extends ScriptBuilder {
     )}})`;
   };
 }
+
+const truncateText = (str: string, maxLen: number) => {
+  return `${str.substring(0, maxLen)}${str.length > maxLen ? '...' : ''}`;
+};
+
+function describeAction(action: Action, lib: ScriptType) {
+  return action?.type === ActionType.Click
+    ? `Click on <${action.tagName.toLowerCase()}> ${
```

Comment:
Not important now but one thing that would be nice is if we can somehow magically consolidate this and the logic here: https://github.com/DeploySentinel/Recorder/blob/e09bf9062ec3f5f9457146cb738b371bd8b2fa92/src/pages/Content/ControlBar.tsx#L57

Though honestly not sure if there's a good pattern for that given this is supposed to be just text and the other stylized tex/JSX

Reply:
Good point! I'm going to consolidate this by moving it into Action context. Open to different ideas. We will see in coming PRs